optuna/integration/lightgbm.py

```diff
@@ -10,28 +12,24 @@
     # LightGBMPruningCallback is disabled because LightGBM is not available.
     _available = False
 
-
 # Attach lightgbm API.
+_names_from_tuners = ['train', 'LGBMModel', 'LGBMClassifier', 'LGBMRegressor']
+
 if _available:
     # API from optuna integration.
     from optuna.integration import lightgbm_tuner as tuner
 
     # Workaround for mypy.
```

Comment:
I do not know the reason why we need do workaound... check it later.

Reply:
I'm not sure, but it might be for mypy check in py2.

Reply:
Oh, sorry. I misunderstood that there was a wrong comment left. Thank you for making the comment clear!